sympy/polys/distributedmodules.py

```diff
@@ -706,7 +712,7 @@ def removethis(pair):
     # Now carry out the buchberger algorithm.
     while P:
         i, j, s, t = P.pop()
-        f, sf, g, sg = S[i], Sugars[i], S[j], Sugars[j]
```

Comment:
/!\ Ssugar, Sugar and Sugars appears in this routine. This needs an audit to see which spelling is needed...I suspect you won't want to delete these lines.

Reply:
sf and sg are not used so from that perspective it should be OK.

Assuming that we had some sort of code quality checker this would be an obvious complaint. Then, as you mention elsewhere (I comment in a, sort of, random order, sorry about that...), it may be that it is "work in progress", in which case this may make it slightly harder to get back on track, or that the code is simply not working because the unused variable should be used, in which case removing it is removing a hint. Still, quite likely, it is a result of either sloppy coding (copy and paste - modifying - no cleanup, seen quite a few of those) or modifying something without cleaning up. Hard to say. Some time I have checked the history to make a more educated guess, sometimes not.

Reply:
> sf and sg are not used so from that perspective it should be OK.

ok